ci-operator/infra/ansible-runner-imagestream.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  name: ansible-runner
+  namespace: openshift
+spec:
```

Comment:
Do you want to add a scheduled import here to auto-update the imagestream from quay?